MyBlog.Services/Areas/Users/UserService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore;
+using MyBlog.Infrastructure;
+using MyBlog.Infrastructure.Entities;
+using MyBlog.Service.Exception;
+
+namespace MyBlog.Service.Areas.Users;
+
+public class UserService : IUserService
+{
+    private readonly MyBlogContext _db;
+
+    public UserService(MyBlogContext db)
+    {
+        _db = db;
+    }
+
+    public async Task<List<User>> GetAllAsync()
+    {
+        return await _db.Users.ToListAsync();
```

Comment:
win the case with small changes, you can write it by one line, but in the future, you will have over 15-20 lines in the method, so i recomend you write easy-readeble code, for example:
var users = await _context.Users.ToListAsync();

return user;

apply this style to all methods in your code(if you change something, you should have only one code-style in your code)